docs/reference/indices/split-index.asciidoc

```diff
@@ -52,9 +103,9 @@ Splitting works as follows:
 * Finally, it recovers the target index as though it were a closed index which
   had just been re-opened.
 
```

Comment:
Like with shrinking, referring to splitting as "it" is a bit odd. If you make changes there, I'd make parallel changes here.

Reply:
Fixed with 6f981b96518da9cef9c3df87e5a546303f42b0f6.